llpc/test/shaderdb/core/LocalInvocationIdSwizzle.spvasm

```diff
@@ -1,108 +0,0 @@
-; BEGIN_SWIZZLETEST
```

Comment:
Is this test deleted on purpose or is it missing in the internal repo?

The rest looks good to me.

Reply:
Many thanks for pointing it out, added now.